package.json

```diff
@@ -24,7 +24,7 @@
     "@types/lodash": "^4.14.121",
     "@types/node": "^10.12.9",
     "@types/node-fetch": "^2.1.3",
-    "danger": "^7.0.9",
+    "danger": "^7.1.3",
```

Comment:
Is the danger upgrade required for this PR or is it just a drive-by change?

Reply:
just drive-by; would you recommend leaving it out?

Reply:
I think it's fine to leave in, but maybe split it out into its own commit if unrelated to other changes? Happy to help out with that as it can be a bit tricky.